components/service/glean/docs/pings/custom.md

```diff
@@ -3,9 +3,30 @@
 Applications can define metrics that are sent in custom pings. Unlike the
 built-in pings, custom pings are sent explicitly by the application.
 
+## Defining a custom ping
+
+Custom pings must be defined in a `pings.yaml` file, which is in the same
+directory alongside your app's `metrics.yaml` file.
+
+Each ping has the following parameters:
+
+- `description` (required): A human-readable description of the ping.
+- `include_client_info` (required): A boolean indicating whether to include the
+  [`client_info`](pings.md#The-client_info-section)) section.
+- `disabled` (optional): When `true`, sending of the ping is disabled.
+
+For example, to define a custom ping specifically for search information:
+
+```YAML
+search:
+  description: >
+    A ping to record search data.
+  include_client_info: false
```

Comment:
We don't want to exclude the full `client_info`, as this also contains informations about the app version, channel and other metadata which are not PII. As decided [here](https://docs.google.com/document/d/1dGiUmR63Kd1oqoL7eQzEana0gadPffvq00LEith3uCY/edit#), we only want to allow excluding the `client_id`.